draft-ietf-quic-transport.md

```diff
@@ -2526,6 +2571,9 @@ QUIC_CLOSED_CRITICAL_STREAM (0x80000007):
 QUIC_MISSING_PAYLOAD (0x80000030):
 : The packet contained no payload.
 
+QUIC_RECEIVED_RST (0x80000035):
+: Terminating stream because peer sent a RST_STREAM or DISINTEREST.
```

Comment:
Having just gone over the error codes, I think that NO_ERROR is perfectly appropriate for this.  You have removed the existing one-way semantic from RST_STREAM+NO_ERROR and put that in a new frame, but the idea remains that there are cases for RST_STREAM where the stream just isn't wanted any more.  NO_ERROR fits perfectly to my mind.